Day-II-Calculator/calculator/src/App.css

```diff
@@ -0,0 +1,23 @@
+.app-wrapper {
+  max-width: 452px;
+  margin: 0 auto;
+  
```

Comment:
This is a very minor issue, but there is an extra space here that you may want to remove so the spacing throughout your code is consistent.